util.py

```diff
@@ -209,22 +209,23 @@ def parse_nonlinearity(nonlinearity_str):
 # FIXME: tasks are imported at this point to avoid a circular dependency:
 # tasks [*] -> models [SLClassifier] -> util [parse_nonlinearity]
 from tasks import *
+import tasks
 
 def get_task_by_name(args, task_name):
     if 'Offenseval' == task_name:
-        return OffensevalTask(cls_dim=args.mlp_dims[-1])
+        return tasks.OffensevalTask(cls_dim=args.mlp_dims[-1])
```

Comment:
Now we import tasks twice (from tasks import *, and import tasks). Are you sure that this way we do not have problems with circular imports/ Are there no other functions/classes of tasks used in the file?

Reply:
Originally it just said from tasks import * and it didn't work for me so I added the other import. Now it works. Hopefully it doesn't hurt ;)

Reply:
It should work both ways, I might create another commit to remove the "from tasks import *" later.